app/experimenter/normandy/tests/test_tasks.py

```diff
@@ -335,6 +335,39 @@ def test_live_rollout_updates_population_percent(self):
         experiment = Experiment.objects.get(normandy_id=1234)
         self.assertEqual(experiment.population_percent, decimal.Decimal("50.000"))
 
+    def test_firefox_version_updates(self):
+        experiment = ExperimentFactory.create(
+            status=Experiment.STATUS_LIVE,
+            normandy_id=1234,
+            firefox_min_version="80.0",
+            firefox_max_version="80.0",
+        )
+
+        mock_response_data = {
+            "approved_revision": {
+                "enabled": True,
+                "filter_object": [
+                    {"type": "version", "versions": [79.0, 80.0, 81.0, 82.0]}
```

Comment:
Do versions come in from normandy as floats like this? or as strings?

Reply:
ooh you're right versions are listed at integers from normandy